package.json

```diff
@@ -27,12 +27,13 @@
     "aws-sdk": "2.1376.0",
     "axios": "^1.4.0",
     "cfenv": "1.2.4",
-    "connect-dynamodb": "3.0.0",
+    "connect-dynamodb": "2.0.5",
```

Comment:
Ooh! What happened here?

Reply:
This was out of sync with some of the other libraries we are using causing a failure, so I reverted it.